### DISCUSSION TITLE
How can I stop my Select options getting reset when visibility toggled

### DISCUSSION THREAD

**Discussion by bsamworthdev** [filamentphp/filament#10383] [Help]
I've got a form with a toggle and a select. When the toggle is turned off the select should be hidden. But when I toggle back on the select has its options removed. 

How can I stop this happening?

Section::make('Contact Address')
                        ->schema([
                            Toggle::make('useSameAddress')
                                ->label('Same as residential address')
                                ->live()
                                ->reactive(),
                            Grid::make(1) 
...
                                ->schema([
                                    Select::make('contactcountry')
                                        ->extraAttributes(['class' => 'contact_country'])
                                        ->placeholder('Select option')
                                        ->searchable()
                                        ->options($this->countries)
                                        ->label('Country'),
                                    ])
                                ->hidden(
                                        fn (Get $get): bool => $get('useSameAddress') === true
                                    )
                            ]),

**Comment by leandrocfe** [comment]
*When the toggle is turned off the select should be hidden*
`$get('useSameAddress') === false` actually

Where is `$this->countries` ? Probably you have an issue here.
Did you try it with static options?

```php
->options([
    'US' => 'United States',
    'CA' => 'Canada'
])
```

`live()` and `reactive()` are the same

**Reply by bsamworthdev** [reply]
Yeah good shout.

When I set the countries property in the mount method of the component I still get the same issue:
$this->countries = [
            'US' => 'United States',
            'CA' => 'Canada'
        ];

...but when I just pass the array directly in the issue goes away.
->options([
            'US' => 'United States',
            'CA' => 'Canada'
        ])
